src/pyhf/tensor/pytorch_backend.py

```diff
@@ -11,12 +11,21 @@ class pytorch_backend(object):
 
     def __init__(self, **kwargs):
         self.name = 'pytorch'
+        self._mode = 'GPU'
         self.dtypemap = {
             'float': getattr(torch, kwargs.get('float', 'float32')),
             'int': getattr(torch, kwargs.get('float', 'int32')),
             'bool': torch.bool,
         }
 
+    @property
+    def mode(self):
+        return self._mode
+
+    @mode.setter
```

Comment:
To make this easier to deal with -- we shouldn't allow the mode to be changed after initialization. If the user wants to change the mode, they should supply a new backend...